pkg/storage/cache.go

```diff
@@ -1,297 +0,0 @@
-package storage
```

Comment:
why delete this file here?

Reply:
The file contained the boltdb implementation, which was moved to its own folder